src/main/java/org/reaktivity/nukleus/sse/internal/SseConfiguration.java

```diff
@@ -45,4 +51,8 @@ public DirectBuffer initialComment()
         return SSE_INITIAL_COMMENT_ENABLED.getAsBoolean(this) ? INITIAL_COMMENT_DEFAULT : null;
     }
 
+    public StringFW getEventType() {
+        return new StringFW(EVENT_TYPE.get(this));
+    }
```

Comment:
This should return `String` rather than `StringFW` since otherwise each call to this method creates garbage.